uncrustify.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+#
+#   Copyright (c) 2016 Jack R. Dunaway. All rights reserved.
+#
+#   Permission is hereby granted, free of charge, to any person obtaining a copy
+#   of this software and associated documentation files (the "Software"),
+#   to deal in the Software without restriction, including without limitation
+#   the rights to use, copy, modify, merge, publish, distribute, sublicense,
+#   and/or sell copies of the Software, and to permit persons to whom
+#   the Software is furnished to do so, subject to the following conditions:
+#
+#   The above copyright notice and this permission notice shall be included
+#   in all copies or substantial portions of the Software.
+#
+#   THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+#   IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+#   FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+#   THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#   LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#   FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS
+#   IN THE SOFTWARE.
+#
+
+find -E ./perf ./src ./tests ./tools -iregex ".*\.(h|c|inc)" -print0 |
+    xargs -0 uncrustify -l C -c .uncrustify.cfg --replace --no-backup
```

Comment:
Btw, not every platform has a working -print0 and xargs -0.  I added those to illumos fairly recently, and I don't think Solaris has it.  Since our files don't have spaces in their names, I think we can use just ordinary -print and no -0 for xargs.